build/Dockerfile

```diff
@@ -1,10 +1,12 @@
 FROM ansible/ansible-runner
 
+RUN yum remove -y ansible
+
 RUN pip install --upgrade setuptools
-RUN pip install urllib3==1.23
-RUN pip install openshift==0.6.0 kubernetes==6.0.0 ansible-runner-http
+RUN pip install idna~=2.7 ansible openshift kubernetes ansible-runner-http
```

Comment:
I still think removing `ansible-runner` + pip installing it is probably the right thing to do, the `idna` requirement made it fail to start but I worry about other mismatched dependencies that aren't so explosive.